src/explore-education-statistics-admin/src/pages/release/components/ReleaseSummaryForm.tsx

```diff
@@ -102,7 +102,13 @@ const ReleaseSummaryForm = <
 
   const baseSchema = Yup.object<ReleaseSummaryFormValues>({
     timePeriodCoverageCode: Yup.string().required('Choose a time period'),
-    timePeriodCoverageStartYear: Yup.string().required('Enter a year'),
+    timePeriodCoverageStartYear: Yup.string()
+      .required('Enter a year')
+      .test(
+        'length',
```

Comment:
No need for a custom validator, there's literally a `length` method built into `Yup.string`.